### ISSUE TITLE
libpango-1.0.so.0: undefined symbol: g_log_structured_standard

### ISSUE THREAD

**Issue by probonopd** [AppImage/appimaged#82]
A user reported on IRC:

`[user@fedora-29 ~]$ ./appimaged-x86_64.AppImage --install ./appimaged-x86_64.AppImage: symbol lookup error: /lib64/libpango-1.0.so.0: undefined symbol: g_log_structured_standard`

Why are we depending on libpango?

**Comment by azubieta** [comment]
`libcairo` depends on it. If we get rid of the image processing features this will be fixed too.

**Comment by showcheap** [comment]
Sampe problem when run appimaged on Fedora Workstation 30

```
NAME=Fedora
VERSION="30 (Workstation Edition)"
ID=fedora
VERSION_ID=30
VERSION_CODENAME=""
PLATFORM_ID="platform:f30"
PRETTY_NAME="Fedora 30 (Workstation Edition)"
ANSI_COLOR="0;34"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:30"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f30/system-administrators-guide/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=30
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=30
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Workstation Edition"
VARIANT_ID=workstation
```

**Comment by probonopd** [comment]
This probably affects all Fedora versions newer than 29, and possibly all other very recent distributions as well.

**Comment by probonopd** [comment]
Can we statically link everything that is needed, instead of using `dlopen()`

**Comment by probonopd** [comment]
@showcheap could you please test the version linked in https://github.com/AppImage/appimaged/pull/86? It should hopefully fix this issue.

**Comment by showcheap** [comment]
@probonopd @azubieta It's works :+1: :smile: 


![Screenshot from 2019-05-21 13-05-03](https://user-images.githubusercontent.com/1310895/58072302-739fb480-7bc9-11e9-8dc3-1eda44b0a62f.png)

**Comment by probonopd** [comment]
Great, thanks for testing @showcheap 💯

**Comment by azubieta** [comment]
This should be fixed by #87, will close the issue. Please reopen if required.
